change_notes/2023-10-04-aggregate-literals-from-variadic-templates.md

```diff
@@ -0,0 +1 @@
+ * `M8-5-2` - recognise aggregate literals initialized with parameters from variadic templates.
```

Comment:
Applying the style for changenotes from our [handbook](https://github.com/github/codeql-coding-standards/blob/main/docs/development_handbook.md#change-notes)
```suggestion
 `M8-5-2` - `AggregateLiteralEnhancements.qll`:
   - recognise aggregate literals initialized with parameters from variadic templates.
```